.gitignore

```diff
@@ -4,3 +4,6 @@
 coverage
 /test/fixtures/certs/tmp
 accept.json
+
+# IDE
+.idea
```

Comment:
You could use a global gitignore for your IDE files: http://egorsmirnov.me/2015/05/04/global-gitignore-file.html

Reply:
Fair enough! Thanks!